src/apps/media/src/app-revamp/components/Thumbnail/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+import { CardMedia, Card, Typography, CardContent } from "@mui/material";
+import { fileExtension } from "../../utils/FileUtils";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFile } from "@fortawesome/free-solid-svg-icons";
+
+interface ThumbnailProps {
+  src?: string;
+  file: any;
+  width?: number;
+  height?: number;
+}
+
+export const Thumbnail: FC<ThumbnailProps> = ({ src, file, width, height }) => {
+  const CardStyle = {
+    maxWidth: width,
+    maxHeight: height,
+    borderWidth: "1px",
+    borderColor: "grey.100",
+    borderStyle: "solid",
+  };
+
+  const CardContentStyle = {
+    px: 1,
+    overflow: "hidden",
+    whiteSpace: "nowrap",
+    textOverflow: "ellipsis",
+  };
+
+  switch (fileExtension(file.url)) {
+    case "jpg":
+    case "jpeg":
+    case "png":
+    case "gif":
+    case "svg":
+    case "webp":
+      return (
+        <Card sx={CardStyle} elevation={0}>
+          <CardMedia
+            component="img"
+            data-src={src}
+            image={src}
```

Comment:
The one you're stating is like this one right?
![image](https://user-images.githubusercontent.com/40680430/190042725-87044d37-b036-4230-9e0d-b2c63285bd90.png)

Reply:
@agalin920 Is there a way to call the CONFIG.SERVICE_MEDIA_RESOLVER in ts files? Because currently that variable is inside app.config.js that serves the different instances.